kani-driver/src/args.rs

```diff
@@ -535,6 +539,14 @@ impl KaniArgs {
 
         Ok(())
     }
+
+    fn print_deprecated(&self, option: &str) {
+        if !self.quiet {
+            warning(&format!(
+                "The `{option}` flag is deprecated and it will be removed in future versions"
```

Comment:
I would try to state outright that it does nothing anymore and should be removed.